### ISSUE TITLE
No Color on negative bars when barGrouping is stacked

### ISSUE THREAD

**Issue by vanarebane** [gitbrent/PptxGenJS#343]
Hello,

I get white boxes on negative values when barGrouping is set to stacked.

Edit: All BAR graphs seem to have white negative colors.
`chartColors `and `invertedColors` will change the whole stack to the same color.

pptxgenjs ver: 2.2.0-beta.20180515

![image](https://user-images.githubusercontent.com/4301343/40834610-00752e22-659a-11e8-9d39-4f2f78be44fa.png)

Code used:
```javascript
`slide.addChart([
        {
            type: pptx.charts.BAR,
            data: [
                {
                    name: 'High',
                    labels: ['Tax', 'Max', 'Mix'],
                    values: [3, 1, 2]
                },
                {
                    name: 'Min',
                    labels: ['Tax', 'Max', 'Mix'],
                    values: [1, 2, 1]
                },
                {
                    name: 'Low',
                    labels: ['Tax', 'Max', 'Mix'],
                    values: [-1, -2, -1]
                }
            ]
        }
    ],
    {
        x:0.5, y:0.5, w:'90%', h:'90%',
        barGrouping: 'stacked',
    }
);`
```

**Comment by octy40** [comment]
@gitbrent - I just encountered this issue as well and I was going to try and see if I can fix and submit a PR. Is there any pointers on where one would start? I tried trace the code, but I can't seem to figure out where the chart bars get added to the slide. Any help is appreciated.

**Comment by octy40** [comment]
@gitbrent - You probably know this already, but the negative values get inverted because of the missing invertIfNegative in the **strXML**. I was able to get it working by adding `strXml += '  <c:invertIfNegative val="0"/>'` on line 2628 in pptxgen.js.

Should this be defined in options? If so I can submit a PR with the change.

**Comment by gitbrent** [comment]
@octy40 

Sure, if this issue can be fixed with a new XML element, please submit a PR along with a test case, and a screencap.

**Comment by gitbrent** [comment]
Thanks @octy40 !

Fixed via Issue #419
